### DISCUSSION TITLE
Wasm benchmarks on Windows?

### DISCUSSION THREAD

**Discussion by timcassell** [dotnet/BenchmarkDotNet#2675] [Q&A]
I'm trying to run the wasm test on Windows 10. I installed all the pre-requisites and disabled the windows check in the toolchain, but I'm still running into errors.

> System.ComponentModel.Win32Exception : An error occurred trying to start process 'v8' with working directory 'C:\BenchmarkDotNet\tests\BenchmarkDotNet.In
      tegrationTests\bin\Release\net8.0\BenchmarkDotNet.IntegrationTests-Dry-1\bin\Release\net8.0\browser-wasm\AppBundle'. The system cannot find the file specified.

Not sure why that's happening as the v8 is in my PATH and I can run it directly in cmd. So anyway, I passed the explicit path to v8 `C:\Users\Tim\.jsvu\bin\v8.cmd`, and that managed to get past that error. But then the benchmark failed:

> Warning: unknown flag --expose_wasm.
Try --help for options
No Workload Results were obtained from the run.
// Benchmark Process 17404 has exited with code 1.

And that's where I got stuck. Any ideas?

**Comment by adamsitnik** [comment]
The last time I've tried it (16/07/2020!), it worked only on Unix (https://benchmarkdotnet.org/articles/configs/toolchains.html#wasm).

The dotnet/runtime docs don't mention Windows at all: https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/libraries/testing-wasm.md

Perhaps it's by design? cc @radical @caaavik-msft
